libcalico-go/lib/validator/v3/validator.go

```diff
@@ -45,6 +45,9 @@ const (
 	// Maximum size of annotations.
 	totalAnnotationSizeLimitB int64 = 256 * (1 << 10) // 256 kB
 
+	// linux can support route-tables with indices up to 0xfffffff, however, using all of them would likely blow up, so cap the limit at 65535
```

Comment:
How about capping the size of a single _range_ but allowing that range to be placed anywhere in the 32-bits?  That seems like a happy medium.

Reply:
nice, sounds good